.circleci/config.yml

```diff
@@ -1,8 +1,47 @@
-version: 2
+version: 2.1
+
+commands:
+  dktl:
+    steps:
+      - run:
+          name: Install/configure DKAN Tools (dktl)
+          command: |
+            if [ ! -d ~/dkan-tools ]; then git clone -b $DKTL_VERSION https://github.com/GetDKAN/dkan-tools.git ~/dkan-tools; fi
+            export PATH=~/dkan-tools/bin:$PATH
+            echo "export PATH=~/dkan-tools/bin:$PATH" >> $BASH_ENV
+  prepare_site:
+    parameters:
+      upgrade:
+        description: "If true, will upgrade a stable instance to the target commit"
+        default: false
+        type: boolean
+    steps:
+      - run:
+          name: Install Drupal
+          command: |
+            dktl dc up -d
+            sleep 10 && dktl install
+      - when:
+          condition: << parameters.upgrade >>
+          steps:
+            - run:
+                name: Switch to target commit and run DKAN upgrade
+                command: |
+                  git --git-dir=dkan/.git --work-tree=dkan checkout "${CIRCLE_SHA1}"
+                  dktl exec:composer -- require 'getdkan/dkan:dev-HEAD' --no-update --ansi
+                  dktl make
```

Comment:
This may be counterintuitive but the idea of `dktl make` is you only run it once. It uses `composer install` and what you want here is an update I believe. I think this should change to just `dktl composer update`.

Reply:
Updated